src/quicktype-core/language/Python.ts

```diff
@@ -115,12 +115,13 @@ export const pythonOptions = {
         ],
         "3.6"
     ),
-    justTypes: new BooleanOption("just-types", "Classes only", false)
+    justTypes: new BooleanOption("just-types", "Classes only", false),
+    nicePropertyNames: new BooleanOption("nice-property-names", "Transform property names to be pythonic", false),
```

Comment:
I believe "Pythonic" is capitalized, according to [the Python style guide](https://docs.python-guide.org/writing/style/).